packages/core/accounts/kernel/utils/plugins/multi-chain/getMultiUserOpDummySignature.ts

```diff
@@ -0,0 +1,51 @@
+import MerkleTree from "merkletreejs"
+import { type UserOperation, getUserOperationHash } from "permissionless"
+import type {
+    EntryPoint,
+    GetEntryPointVersion
+} from "permissionless/types/entrypoint"
+import { type Hex, concatHex, encodeAbiParameters, keccak256 } from "viem"
+
+export const getMultiUserOpDummySignature = <entryPoint extends EntryPoint>(
```

Comment:
I think we can move this action to multi chain package. Since it seems like it requires additional installation of `merkletreejs` package which adds extra load for users not utilising multi chain feature. And this package is already required in multi chain package anyway.